2022-09-20 22:57:02,896 ERROR 9644 [-/127.0.0.1/-/175ms POST /default/addComment] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'articleId' in 'field list'
    at Query.Sequence._packetToError (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    --------------------
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:65:19)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:54:5
    sql: INSERT INTO `comments`(`userId`, `articleId`, `connent`, `addTime`, `like`) VALUES(1, 2, '22', '2022-09-20 22:57', 0)
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'articleId' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `comments`(`userId`, `articleId`, `connent`, `addTime`, `like`) VALUES(1, 2, '22', '2022-09-20 22:57', 0)"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 9644
hostname: hahas-MBP.lan

2022-09-20 22:59:36,090 ERROR 9644 [-/127.0.0.1/-/149ms POST /default/addComment] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'connent' in 'field list'
    at Query.Sequence._packetToError (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    --------------------
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:65:19)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:54:5
    sql: INSERT INTO `comments`(`userId`, `article_id`, `connent`, `addTime`, `like`, `userName`) VALUES(1, 2, '22', '2022-09-20 22:59', 0, 'j')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'connent' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `comments`(`userId`, `article_id`, `connent`, `addTime`, `like`, `userName`) VALUES(1, 2, '22', '2022-09-20 22:59', 0, 'j')"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 9644
hostname: hahas-MBP.lan

2022-09-20 23:00:06,363 ERROR 9644 [-/127.0.0.1/-/146ms POST /default/addComment] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'like' in 'field list'
    at Query.Sequence._packetToError (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    --------------------
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:65:19)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:54:5
    sql: INSERT INTO `comments`(`userId`, `article_id`, `content`, `addTime`, `like`, `userName`) VALUES(1, 2, '22', '2022-09-20 23:00', 0, 'j')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'like' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `comments`(`userId`, `article_id`, `content`, `addTime`, `like`, `userName`) VALUES(1, 2, '22', '2022-09-20 23:00', 0, 'j')"
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 9644
hostname: hahas-MBP.lan

